src/components/invite/Invite.js

```diff
@@ -111,6 +111,15 @@ const ShareBox = ({ level }) => {
     abTestMessage,
   ])
 
+  const shareUrlStyle = useMemo(
+    () =>
+      Platform.select({
```

Comment:
you could definer shareLink in styles via Platform.select as it doest depends on the component props/state
so you don't need to use memo here. just define in getStylesFromProps

Reply:
ok